subprojects/docs/src/docs/userguide/organizingGradleProjects.adoc

```diff
@@ -15,55 +15,122 @@
 
-++++
-<sample xmlns:xi="http://www.w3.org/2001/XInclude" id="configureProjectUsingScript" dir="userguide/organizingGradleProjects/configureProjectUsingScript" title="Configuring the project using an external build script">
-    <sourcefile file="build.gradle"/>
-    <sourcefile file="other.gradle"/>
-    <output args="-q hello"/>
-</sample>
-++++
+----
+.
+├── build.gradle
+├── settings.gradle
+└── src
+    └── main
+        ├── java
+        │   └── HelloWorld.java
+        └── kotlin
+            └── Utils.kt
+----
 
-[[sec:build_sources]]
-=== Build sources in the `buildSrc` project
+[[sec:use_standard_conventions]]
+=== Use standard conventions as much as possible
+
+All Gradle core plugins follow the software engineering paradigm link:https://en.wikipedia.org/wiki/Convention_over_configuration[convention over configuration]. The plugin logic provides users with sensible defaults and standards, the conventions, in a certain context. Let’s take the <<java_plugin,Java plugin>> as an example.
+
+* It defines the directory `src/main/java` as the default source directory for compilation.
+* The output directory for compiled source code and other artifacts (like the JAR file) is `build`.
 
-When you run Gradle, it checks for the existence of a directory called `buildSrc`. Gradle then automatically compiles and tests this code and puts it in the classpath of your build script. You don't need to provide any further instruction. This can be a good place to add your custom tasks and plugins.
+By sticking to the default conventions, new developers to the project immediately know how to find their way around. While those conventions can be reconfigured, it makes it harder to build script users and authors to manage the build logic and its outcome. Try to stick to the default conventions as much as possible except if you need to adapt to the layout of a legacy project. Refer to the reference page of the relevant plugin to learn about its default conventions.
 
-For multi-project builds there can be only one `buildSrc` directory, which has to be in the root project directory.
+=== Always define a setting file
 
-Listed below is the default build script that Gradle applies to the `buildSrc` project:
+Gradle tries to locate a `settings.gradle` (Groovy DSL) or a `settings.gradle.kts` (Kotlin DSL) file with every invocation of the build. For that purpose, the runtime walks the hierarchy of the directory tree up to the root directory. The algorithm stops searching as soon as it finds the settings file.
 
+Always add a `settings.gradle` to the root directory of your build to avoid the initial performance impact. This recommendation applies to single project builds as well as multi-project builds. The file can either be empty or define the desired name of the project.
+
+A typical Java-based project could look as such:
```

Comment:
I think you can omit "Java-based" in accordance with the next bit of feedback

Reply:
done